addons/udes_sale_stock/models/courier.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+class UdesCourier(models.Model):
+    _name = "udes.courier"
+    _description = "Courier"
```

Comment:
Let's keep this separate to our existing implementations of `udes.courier` and call this `udes.carrier`

Reply:
So change all instances of "courier" to "carrier" please